OpenSim/Common/Test/testDataTable.cpp

```diff
@@ -76,11 +76,16 @@ int main() {
                 throw Exception{"Test failed: "
                                 "table.getColumnIndex(labels.at(i)) != i"};
     }
-    // Print out the DataTable to console.
-    try {
-        std::cout << table << std::endl;
-        throw Exception{"Test failed: Exception expected."};
-    } catch(const OpenSim::EmptyTable&) {}
+
+    // Test exceptions (table should be empty here).
+    SimTK_TEST_MUST_THROW_EXC(table.getRowAtIndex(0),
```

Comment:
Same fix as noted above. This `RowIndexOutOfRange` is thrown from [`DataTable_::getRowAtIndex()`](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Common/DataTable.h#L674).